ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -989,6 +989,7 @@ wan_virtual_topologies:
 | AvdTestBGP | VerifyBGPSpecificPeers | Validate the state of BGP Address Family sessions, including `Path-Selection` for AutoVPN, `Link-State` and `IPv4/IPv6 SR-TE` for CV Pathfinder. |
 | AvdTestIPSecurity | VerifySpecificIPSecConn | Validate the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
 | AvdTestStun | VerifyStunClient | Validate the presence of a STUN client translation for a given source IPv4 address and port. The list of expected translations for each device is built by searching local interfaces in each path-group. |
+| AvdTestDpsReachability | VerifyReachability | Validate the DPS to DPS interface reachability. |
```

Comment:
You need to update the `eos_validate_state` doc as well --> https://avd.arista.com/devel/roles/eos_validate_state/index.html